### PR TITLE
#161807971 create a comprehensive readme documentation

### PR DESCRIPTION
**Pull request by johngorithm**
#### What does this PR do?
create a comprehensive readme documentation

#### Tasks needed to be completed
- update readme file with info about the project

#### Pivotal Tracker ID associated with this story
161807971

#### Screenshots
screen shot 2018-11-08 at 5 18 19 pm

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,80 @@
...
+* clone this repository
+* switch into the project folder
+* install dependencies by running `npm install` on your *command-line* opened in the project's root directory
+* run `npm test` run test
+
+
+
+## Deployment
+* Any pull request raised on this project is automatically deployed to **Heroku** as a preview app. The link to this app is accessible via the PR's page.
+* The **develop** branch automatically deploys whenever ever a pull request is merged.
```
Comment (joeeasy) [inline README.md]: Nice job on the readme,  however, you mistakenly added `ever` to after `whenever`

File: README.md [hunk 2]
```diff
@@ -1,6 +1,80 @@
...
+## Metis-ah-frontend 
+[![CircleCI](https://circleci.com/gh/andela/metis-ah-frontend/tree/develop.svg?style=svg)](https://circleci.com/gh/andela/metis-ah-frontend/tree/develop) [![Test Coverage](https://api.codeclimate.com/v1/badges/93530611b5b0d23beebf/test_coverage)](https://codeclimate.com/github/andela/metis-ah-frontend/test_coverage) [![Build Status](https://travis-ci.org/andela/metis-ah-frontend.svg?branch=develop)](https://travis-ci.org/andela/metis-ah-frontend)
+
+
+## Description
+
+This project creates a platform for Authors to create and share articles. It is equally a platform that allows user to read articles, react to them and socialize with one another.
+
+### Website url
+https://metis-ah-frontend.herokuapp.com/
```
Comment (joeeasy) [inline README.md]: Well done for providing the link to viewing the application, you could also use a markdown like this ```[View live app Here](https://metis-ah-frontend.herokuapp.com/)``` [View live app Here](https://metis-ah-frontend.herokuapp.com/)

**Review by joeeasy — CHANGES_REQUESTED** [review]
Good job John, just fix the tiny changes 👍🏻

File: README.md [hunk 3]
```diff
@@ -1,6 +1,80 @@
...
 Create a community of like minded authors to foster inspiration and innovation
 by leveraging the modern web.
+
+## Metis-ah-frontend 
+[![CircleCI](https://circleci.com/gh/andela/metis-ah-frontend/tree/develop.svg?style=svg)](https://circleci.com/gh/andela/metis-ah-frontend/tree/develop) [![Test Coverage](https://api.codeclimate.com/v1/badges/93530611b5b0d23beebf/test_coverage)](https://codeclimate.com/github/andela/metis-ah-frontend/test_coverage) [![Build Status](https://travis-ci.org/andela/metis-ah-frontend.svg?branch=develop)](https://travis-ci.org/andela/metis-ah-frontend)
+
+
+## Description
+
+This project creates a platform for Authors to create and share articles. It is equally a platform that allows user to read articles, react to them and socialize with one another.
```
Comment (emmaadesile) [inline README.md]: Good job on the readme. Please check the last part of line 14, user should be users

File: README.md (hunk 1 above)
Comment (johngorithm) [reply README.md]: Alright.
Thanks for the observation

**Review by emmaadesile — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (johngorithm) [reply README.md]: Thanks.
I will sure look into that

**Review by emmaadesile — APPROVED** [review]

**Review by joeeasy — APPROVED** [review]

**Pull request merged**